Assignment1/Assignment1_QNo6/SolAssignmentNo1_QNo6.rb

```diff
@@ -0,0 +1,25 @@
+# =>  6. Find out how to convert the string 'The Ruby Book is based on Ruby'
+# => to 'The Rails Book is based on Rails' ?
+# => author : Hrisikesh Singha Chowdhury
+
+# => Fn to implement buisness logic
+def replace_word
+	# => Repeateadly asking users to give correct input strings
+	begin
+		puts "Enter the sentence"
+		sent = gets.chomp
+  end until (sent != nil)
+  puts "Enter the old word to be replaced"
+	old_word = gets.chomp
+	puts "Enter the new word by which to be replaced"
+	new_word = gets.chomp
+	new_sent = sent.gsub!(old_word,new_word)
+	if (new_sent != nil)
```

Comment:
@HrisikeshSinghaChowdhury There is an inbuilt method `nil?` Can you use that here?

Reply:
yes